### ISSUE TITLE
Add a link build scheme somewhere in README?

### ISSUE THREAD

**Issue by ssuukk** [idolpx/meatloaf#5]
I see you mentioning buildging my own Meatloaf server. But it's no use without a Meatloaf. Is there a scheme to build it anywhere?

**Comment by idolpx** [comment]
It is just some PHP scripts that you host on any HTTP server with PHP support.  Just ftp the scripts over and upload some files to host.  Then have meatloaf load the URL to browse through the server.

**Comment by ssuukk** [comment]
Yeah, but how am I supposed to get a meatloaf?!

pon., 3 maj 2021, 18:36 użytkownik idolpx ***@***.***>
napisał:

**Comment by idolpx** [comment]
Have to build one at the moment.  It's not difficult.  You don't even need the level shifter actually.  I've tested without it and it works fine.  Bare minimum that you need is a WEMOS D1 Mini (4MB or 16MB) or NodeMCU and a DIN6 male connector and wire.  I would recommend getting an ESP32 though.  I'm working with the FujiNet guys to integrate the Meatloaf functionality into their project and the ESP32 is much more powerful.  :)

https://www.aliexpress.com/item/32883116057.html

That dev board is pretty great for the price.

If you want the modem emulation functionality you will need the edge card connector for the user port too.
I hope to have more time to work on things in the future and actually develop some custom boards.

**Comment by ssuukk** [comment]
Ok, but is there a building schematics for meatloaf?

pon., 3 maj 2021, 22:08 użytkownik idolpx ***@***.***>
napisał:

**Comment by idolpx** [comment]
yes...  look under docs.   Hit me up on Discord if you have any questions.  :)

https://discord.gg/FwJUe8kQpS

**Comment by ssuukk** [comment]
Ok, thanks, I'll see what I can do with the docs.

pon., 3 maj 2021, 22:15 użytkownik idolpx ***@***.***>
napisał:

**Comment by idolpx** [comment]
I just updated the wiring doc to remove the level shifter.  It is not needed and you don't have to wire up SRQ or RESET either right now.  Those will be used in the future.

**Comment by ssuukk** [comment]
In case I need help - should we continue in this thread, or us there a
better way to contact you?

pon., 3 maj 2021, 22:56 użytkownik idolpx ***@***.***>
napisał:

**Comment by idolpx** [comment]
Yes... login to discord and we can chat there.

https://discord.gg/FwJUe8kQpS
